examples/component/http-client/README.md

```diff
@@ -0,0 +1,83 @@
+# TinyGo HTTP Client
+
+This repository contains a WebAssembly Component written in [TinyGo][tinygo], which:
```

Comment:
Given that TinyGo is the compiler and Go is the language, could we change these to say `Go`?

I think it's fine to include something like `(compiled to WebAssembly with TinyGo)`